index.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Studio Ghibli Creatures</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="creatures.css" />
+</head>
+<body>
+<?php require 'creatures.php'; ?>
+<header><h1>Studio Ghibli Creatures Collection</h1>
```

Comment:
Check indentation in the rest of the file. Remember nested tags are indented below their parent tags!